pkg/loop/config.go

```diff
@@ -164,3 +205,12 @@ func getDatabaseURL() (*url.URL, error) {
 	}
 	return u, nil
 }
+
+// getTelemetryInsecureConnection parses and validates the TRACING_ENABLED environment variable.
+func getTelemetryInsecureConnection() (bool, error) {
```

Comment:
/nit `getTelemetryEnabled` ?

Reply:
Updated, but I just simplified things by removing helpers and in-lining instead for the simple ones.